tests/Snak.tests.js

```diff
@@ -21,11 +21,11 @@ var testSets = [
  * @return {wikibase.datamodel.Snak}
  */
 function constructSnak( SnakConstructor, params ) {
-	return new SnakConstructor( params[0], params[1] );
+	return new SnakConstructor( params[0], params[1], params[2] );
```

Comment:
This is pretty awkward, because this code always passes three constructor arguments, even if there are only two for two of the subclasses. But the test setup was like this before and the change made in this patch doesn't make it worse.